lib/galaxy/config/sample/user_preferences_extra_conf.yml.sample

```diff
@@ -183,3 +183,17 @@ preferences:
               label: URL of the eLabFTW instance you want to access
               type: text
               required: True
+
+    # Used in file_sources_conf.yml
```

Comment:
I guess its fine if we not advertise this option anymore. Instead, we should use the templates. I would remove this addition and the one to the file_sources_conf.yml file